mindjunkies/home/views.py

```
@@ -1,14 +1,51 @@
+    cache_key = f"new_courses_{user_uuid}"
+    new_courses = cache.get(cache_key)
+
+    if new_courses is None:
+        print("Cache miss for new courses")
+        new_courses = Course.objects.filter(status="published", verified=True).order_by("-created_at")[:4]
+        new_courses = list(new_courses)
+        cache.set(cache_key, new_courses, timeout=60 * 5)
+    return new_courses
+
+
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)